app/src/main/res/layout/activity_numeric_input_interaction_view_test.xml

```diff
@@ -49,26 +52,57 @@
       android:focusable="true"
       android:hint="Write here."
       android:inputType="text"
-      android:text="@={textInputViewModel.answerText}"
       android:longClickable="false"
       android:maxLength="200"
       android:padding="8dp"
-      android:singleLine="true" />
+      android:singleLine="true"
+      android:text="@={textInputViewModel.answerText}" />
 
     <org.oppia.app.customview.interaction.FractionInputInteractionView
       android:id="@+id/test_fraction_input_interaction_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_margin="8dp"
       android:background="@drawable/edit_text_background"
-      android:digits="0123456789/- "
-      android:text="@={fractionInteractionViewModel.answerText}"
-      android:focusable="true"
-      android:hint="Write fraction here."
+      android:digits="0123456789/- ."
```

Comment:
. is added for testing purpose.

Reply:
Can you clarify why we need this for testing purposes? If it's for invalid input, maybe just clear the digits to allow free-form text.

Reply:
Okay